LabController/src/bkr/labcontroller/distro_import.py

```diff
@@ -2070,6 +2080,13 @@ def main():
         log_level = logging.INFO
     log_to_stream(sys.stderr, level=log_level)
 
+    if opts.preserve_install_options:
+        if opts.kopts or opts.kopts_post or opts.ks_meta:
+            parser.print_help()
```

Comment:
This doesn't match rest of errors emitted by beaker-import. We are not printing help

Reply:
Done